src/context.ts

```diff
@@ -1,11 +1,6 @@
-import { PrismaClient } from '@prisma/client'
-
-const prisma = new PrismaClient()
-
 export interface Context {
-  prisma: PrismaClient
 }
 
 export const context: Context = {
-  prisma: prisma,
+  //prisma: prisma,
```

Comment:
This cannot be commented out, it is necessary for testing.

Reply:
Can discuss in meeting, but it seems that we can define the Prisma client within index.ts to avoid having to retrieve the context for every resolver.

Reply:
You can just destructure context - no extra effort really